webxrlayers-1.bs

```diff
@@ -898,6 +910,9 @@ The {{XRWebGLBinding}} object is used to create layers that have a GPU backend.
 
   XRWebGLSubImage getSubImage(XRCompositionLayer layer, XRFrame frame, optional XREye eye = "none");
   XRWebGLSubImage getViewSubImage(XRProjectionLayer layer, XRView view);
+
+  readonly attribute FrozenArray&lt;GLenum&gt; supportedColorFormats;
+  readonly attribute FrozenArray&lt;GLenum&gt; supportedDepthStencilFormats;
```

Comment:
👍 Having these as readonly attributes feels better than the getter functions I proposed for the WebGPU bindings.  I'll make the same change there.